_posts/2018-09-03-getting-started/getting-started.Rmd

```diff
@@ -0,0 +1,65 @@
+---
+title: "Getting started with deep learning in R"
+description: |
+  More and more fields are profiting from the use of deep learning, and with the R keras, tensorflow and related packages, you can do deep learning directly in R. In this post, we want to give some orientation as to how to best get started.
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydana2018gettingstarted
+date: 09-03-2018
+categories:
+  - Keras
+output:
+  radix::radix_article:
+    self_contained: false
+preview: images/digits.png
+---
+
+
+There are good reasons to get into deep learning: Deep learning has been outperforming the respective "classical" techniques in areas like image recognition and natural language processing for a while now, and it brings interesting insights even to the analysis of tabular data. For many R users potentially interested in deep learning, the hurdle is not an anticipation of a bit of math creeping up (as many have a background in statistics or empirical sciences). The question is how to get started in an efficient way.
+
+This post will give an overview of materials that should be useful. And if you don't have that background in statistics or similar: Further down, we will also present a few helpful resources to catch up with "the math".
+
+## Keras
+
+The easiest way to get started is using the Keras API. It is a high-level, declarative (in feel) way of specifying a model, training and testing it, originally developed in [Python](http://keras.io) by Francois Chollet and ported to R by JJ Allaire.
+
+Check out the tutorials on the [Keras website](https://tensorflow.rstudio.com/keras/): They introduce basic tasks like classification and regression, as well as basic workflow elements like saving and restoring models, or assessing model performance.
```

Comment:
Maybe we should link to each tutorial here in a bullet list? (not so much b/c we need the direct links but more to flesh out what's available in the tutorials)

Reply:
great idea